styles/mainmenu.sass

```diff
@@ -187,11 +118,12 @@ $main-menu-buttons-button-width: math.div(1, 4)
 	100%
 		transform: translate3d( -2%, 0px, 0px )
 		opacity: 0.0
-	
 
 Label
 	font-size: 25px
 	font-weight: normal
+	font-family: "UniversLTStd"
```

Comment:
Are we shipping this font with the game? At a quick glance it's not in `fonts/`...

Reply:
In Portal 2